core/src/main/java/nl/nn/adapterframework/util/XmlUtils.java

```diff
@@ -279,7 +279,7 @@ public static TransformerPool getRemoveNamespacesTransformerPool(boolean omitXml
 		return getUtilityTransformerPool(xslt,"RemoveNamespaces",omitXmlDeclaration,indent);
 	}
 
-	protected static String makeGetIbisContextXslt() {
+	public static String makeGetIbisContextXslt() {
```

Comment:
This method can be moved to TestPipeline too, and made private there.
getGetIbisContextTransformerPool() can be moved too, and used, or it can be deleted.